index.html

```diff
@@ -21,229 +25,185 @@
                 <li><a class="navlinks" href="#">about</a></li>
                 <li><a class="navlinks" href="#">contact</a></li>
-                </ul>
-            </nav>
+            </ul>
+        </nav>
 
     </header>
 
     <section class="body">
         <div class="head-cont" id="Portfolio">
-             <h1>I'm Emmanuella <br> glad to see  you!</h1>
-
-        <p class="desc">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-
-        <h2>LET'S CONNECT</h2>
-
-        <ul class="social">
-            <li>
-                <a class="social" href="#"><img src="images/Vector.svg" alt="icons"></a>
-            </li>
-            <li>
-                <a class="social" href="#"><img src="images/Linkedin-icon.svg" alt="icons"></a>
-            </li>
-            <li>
-                <a  class="social" href="#"><img src="images/Vector(1).svg" alt="icons"></a>
-            </li>            <li>
-                <a  class="social" href="#"><img src="images/Vector(2).svg" alt="icons"></a>
-            </li>            <li>
-                <a  class="social" href="#"><img src="images/Vector(3).svg" alt="icons"></a>
-            </li>            <li>
-            </li>
-        </ul>
+            <h1>I'm Emmanuella <br> glad to see you!</h1>
+
+            <p class="desc">I'm a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need
+                coded, don't hestiate to contact me.</p>
+
+            <h2>LET'S CONNECT</h2>
+
+            <ul class="social">
+                <li>
+                    <a class="social" href="#"><img src="images/Vector.svg" alt="icons"></a>
+                </li>
+                <li>
+                    <a class="social" href="#"><img src="images/Linkedin-icon.svg" alt="icons"></a>
+                </li>
+                <li>
+                    <a class="social" href="#"><img src="images/Vector(1).svg" alt="icons"></a>
+                </li>
+                <li>
+                    <a class="social" href="#"><img src="images/Vector(2).svg" alt="icons"></a>
+                </li>
+                <li>
+                    <a class="social" href="#"><img src="images/Vector(3).svg" alt="icons"></a>
+                </li>
+                <li>
+                </li>
+            </ul>
         </div>
-        
+
     </section>
-  
 
-    <section class="project-container">
-        <div class="project">  
-            <div class="my-img"> 
-        <img class="mobile-image" src="images/face-crop-port.png" alt="picture identity of a lady">
-        </div>
-        <div class="my-text">
+    <!--for popup-->
+    <div class="popup">
+    <!-- <div class="project1">
```

Comment:
- [ ] Your popup has to stick to the chosen Figma template. Please, ensure that your images fit their container, the text and buttons don't overflow their parent element and the popup container has specific padding to match the reference popup design from the Figma template.

**How looks your popup window**
![image](https://user-images.githubusercontent.com/80612925/175573180-f9ab211a-3eb3-4215-be3d-1975f6f19948.png)

**Figma design**
![image](https://user-images.githubusercontent.com/80612925/175573691-9cc1214d-349a-4d1a-86d9-aad92e3b13b9.png)

**_Hint:  You can use the `object-fit: cover` CSS property for the image and add `overflow-y: scroll` for the popup to contain its child elements without its elements exceeding_**

Reply:
thank you corrected it